opensoar/utilities/geojson_serializers.py

```diff
@@ -0,0 +1,87 @@
+from typing import List
+
+from geojson import GeometryCollection, Point, LineString, Polygon, Feature, FeatureCollection
+from opensoar.utilities.helper_functions import calculate_destination
+
+RED_HEX = "#dd1843"
+
+
+def circle_polygon(lat, lng, radius):
+    from shapely.geometry import Point as ShapelyPoint
+    from pyproj import Transformer
+    from shapely.ops import transform
+
+    local_azimuthal_projection = "+proj=aeqd +R=6371000 +units=m +lat_0={} +lon_0={}".format(
+        lat, lng
+    )
+    wgs84_to_aeqd = Transformer.from_proj('+proj=longlat +datum=WGS84 +no_defs', local_azimuthal_projection)
+    aeqd_to_wgs84 = Transformer.from_proj(local_azimuthal_projection,  '+proj=longlat +datum=WGS84 +no_defs')
+    # Get polygon with lat lon coordinates
+    point_transformed = ShapelyPoint(wgs84_to_aeqd.transform(lng, lat))
+
+    buffer = point_transformed.buffer(radius)
+    circle = transform(aeqd_to_wgs84.transform, buffer)
+
+    # todo: find out why nested list necessary
+    circle_coordinates = [[(lon, lat) for lon, lat in circle.exterior.coords]]
+    return Feature(geometry=Polygon(circle_coordinates))
+
+
+def task_to_geojson_features(task) -> List[dict]:
+    # TODO: plot waypoint lines as lines
+    # TODO: have proper cutouts from sectors
+
+    features = []
+    task_line_coords = []
+    for i, waypoint in enumerate(task.waypoints):
+        lon = waypoint.longitude
+        lat = waypoint.latitude
+        task_line_coords.append((lon, lat))
+
+        features.append(circle_polygon(lat, lon, waypoint.r_max))
+
+        # + 180 because orientation is outward
+        angle1 = (waypoint.orientation_angle + 180 - waypoint.angle_max + 360) % 360
+        angle2 = (waypoint.orientation_angle + 180 + waypoint.angle_max) % 360
+        waypoint_fix = {"lon": waypoint.longitude, "lat": waypoint.latitude}
+        end_of_line1 = calculate_destination(waypoint_fix, waypoint.r_max, angle1)
+        end_of_line2 = calculate_destination(waypoint_fix, waypoint.r_max, angle2)
+
+        line1 = LineString([(waypoint_fix['lon'], waypoint_fix['lat']), (end_of_line1['lon'], end_of_line1['lat'])])
```

Comment:
write own class for geojson objects. trivial dicts/lists

Reply:
decided not too. the dependency is very light and makes it easier to extend/change serializers later